x/superfluid/keeper/internal/events/emit.go

```diff
@@ -0,0 +1,133 @@
+package events
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v10/x/superfluid/types"
+)
+
+func EmitSetSuperfluidAssetEvent(ctx sdk.Context, denom string, assetType types.SuperfluidAssetType) {
+	if ctx.EventManager() == nil {
+		return
+	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		newSetSuperfluidAssetEvent(denom, assetType),
+	})
+}
+
+func newSetSuperfluidAssetEvent(denom string, assetType types.SuperfluidAssetType) sdk.Event {
+	return sdk.NewEvent(
+		types.TypeEvtSetSuperfluidAsset,
+		sdk.NewAttribute(types.AttributeDenom, denom),
+		sdk.NewAttribute(types.AttributeSuperfluidAssetType, assetType.String()),
+	)
+}
+
+func EmitRemoveSuperfluidAsset(ctx sdk.Context, denom string) {
+	if ctx.EventManager() == nil {
+		return
+	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		newRemoveSuperfluidAssetEvent(denom),
+	})
+}
+
+func newRemoveSuperfluidAssetEvent(denom string) sdk.Event {
+	return sdk.NewEvent(
+		types.TypeEvtRemoveSuperfluidAsset,
+		sdk.NewAttribute(types.AttributeDenom, denom),
+	)
+}
+
+func EmitSuperfluidDelegateEvent(ctx sdk.Context, lockId uint64, valAddress string) {
+	if ctx.EventManager() == nil {
+		return
+	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		newSuperfluidDelegateEvent(lockId, valAddress),
+	})
+}
+
+func newSuperfluidDelegateEvent(lockId uint64, valAddress string) sdk.Event {
+	return sdk.NewEvent(
+		types.TypeEvtSuperfluidDelegate,
+		sdk.NewAttribute(types.AttributeLockId, fmt.Sprintf("%d", lockId)),
```

Comment:
```suggestion
		sdk.NewAttribute(types.AttributeLockId, utils.Uint64ToString(lockId)),
```

lets change this for tests too